### ISSUE TITLE
support executing custom commands

### ISSUE THREAD

**Issue by pguedes** [pguedes/gesticle#5]
we currently only support mapping gestures to keys (or combos) via xdo

we could support custom commands also

**Comment by Alsan** [comment]
I was think about this too. What do you think about the impact to the executing performance? Recently, the xdo has been loaded into memory since startup, an extra cost is need when spwm a subprocess.

**Comment by pguedes** [comment]
@Alsan yep it's something to consider. I guess we need to test it... other implementations of gestures typically do that so I guess it won't be too bad but we should test it
